src/lkl/setup.c

```diff
@@ -755,9 +784,15 @@ static void lkl_mount_disk(
 
 static void lkl_mount_root_disk(struct enclave_disk_config* disk)
 {
+    bool use_overlay = false;
```

Comment:
use_overlay is set to false and it is always false. It doesn't change. Is this intentional?

Reply:
See the PR description. I will add an app config option.